CLI panic on transfer when missing `--from` parameter

```
$ neo-go wallet nep17 transfer -w wallet.json -r http://localhost:20332 --to NWxY2dFzfHPf4Y2q6AbsSv2ydxmhTTjvDC --token GAS --amount 1000
panic: address was not set

goroutine 1 [running]:
github.com/nspcc-dev/neo-go/cli/flags.(*Address).Uint160(...)
        github.com/nspcc-dev/neo-go/cli/flags/address.go:52
github.com/nspcc-dev/neo-go/cli/wallet.transferNEP17(0xc00009f080, 0x0, 0x0)
        github.com/nspcc-dev/neo-go/cli/wallet/nep17.go:416 +0xb0f
github.com/urfave/cli.HandleAction(0xca7220, 0xf4f280, 0xc00009f080, 0xc000148400, 0x0)
        github.com/urfave/cli@v1.20.0/app.go:490 +0xc8
github.com/urfave/cli.Command.Run(0xdee59d, 0x8, 0x0, 0x0, 0x0, 0x0, 0x0, 0xdf9d1c, 0x15, 0xe1fd02, ...)
        github.com/urfave/cli@v1.20.0/command.go:210 +0x9e8
github.com/urfave/cli.(*App).RunAsSubcommand(0xc000134d00, 0xc00009edc0, 0x0, 0x0)
        github.com/urfave/cli@v1.20.0/app.go:379 +0x88b
github.com/urfave/cli.Command.startApp(0xdeae69, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0xdfe4e1, 0x19, 0x0, ...)
        github.com/urfave/cli@v1.20.0/command.go:298 +0x81a
github.com/urfave/cli.Command.Run(0xdeae69, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0xdfe4e1, 0x19, 0x0, ...)
        github.com/urfave/cli@v1.20.0/command.go:98 +0x1219
github.com/urfave/cli.(*App).RunAsSubcommand(0xc000134b60, 0xc00009ec60, 0x0, 0x0)
        github.com/urfave/cli@v1.20.0/app.go:379 +0x88b
github.com/urfave/cli.Command.startApp(0xdec490, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xe0a16c, 0x24, 0x0, ...)
        github.com/urfave/cli@v1.20.0/command.go:298 +0x81a
github.com/urfave/cli.Command.Run(0xdec490, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xe0a16c, 0x24, 0x0, ...)
        github.com/urfave/cli@v1.20.0/command.go:98 +0x1219
github.com/urfave/cli.(*App).Run(0xc0001349c0, 0xc0000300e0, 0xe, 0xe, 0x0, 0x0)
        github.com/urfave/cli@v1.20.0/app.go:255 +0x741
main.main()
        command-line-arguments/main.go:18 +0x4b
```
When `--from` is not specified we can take the default address from the wallet.

--- a/cli/nep17_test.go
+++ b/cli/nep17_test.go
@@ -111,10 +111,10 @@ func TestNEP17Transfer(t *testing.T) {
 		"neo-go", "wallet", "nep17", "transfer",
 		"--rpc-endpoint", "http://" + e.RPC.Addr,
 		"--wallet", validatorWallet,
-		"--from", validatorAddr,
 		"--to", w.Accounts[0].Address,
 		"--token", "NEO",
 		"--amount", "1",
+		"--from", validatorAddr,
 	}
 
 	t.Run("InvalidPassword", func(t *testing.T) {
@@ -131,6 +131,33 @@ func TestNEP17Transfer(t *testing.T) {
 	require.NoError(t, err)
 	b, _ := e.Chain.GetGoverningTokenBalance(sh)
 	require.Equal(t, big.NewInt(1), b)
+
+	t.Run("default address", func(t *testing.T) {
+		const validatorDefault = "NbTiM6h8r99kpRtb428XcsUk1TzKed2gTc"
+		e.In.WriteString("one\r")
+		e.Run(t, "neo-go", "wallet", "nep17", "multitransfer",
+			"--rpc-endpoint", "http://"+e.RPC.Addr,
+			"--wallet", validatorWallet,
+			"--from", validatorAddr,
+			"NEO:"+validatorDefault+":42",
+			"GAS:"+validatorDefault+":7")
+		e.checkTxPersisted(t)
+
+		args := args[:len(args)-2] // cut '--from' argument
+		e.In.WriteString("one\r")
+		e.Run(t, args...)
+		e.checkTxPersisted(t)
+
+		sh, err := address.StringToUint160(w.Accounts[0].Address)
+		require.NoError(t, err)
+		b, _ := e.Chain.GetGoverningTokenBalance(sh)
+		require.Equal(t, big.NewInt(2), b)
+
+		sh, err = address.StringToUint160(validatorDefault)
+		require.NoError(t, err)
+		b, _ = e.Chain.GetGoverningTokenBalance(sh)
+		require.Equal(t, big.NewInt(41), b)
+	})
 }
 
 func TestNEP17MultiTransfer(t *testing.T) {

--- a/cli/testdata/wallet1_solo.json
+++ b/cli/testdata/wallet1_solo.json
@@ -16,7 +16,7 @@
         "deployed": false
       },
       "lock": false,
-      "isdefault": false
+      "isdefault": true
     },
     {
       "address": "NUVPACMnKFhpuHjsRjhUvXz1XhqfGZYVtY",

--- a/cli/wallet/nep17.go
+++ b/cli/wallet/nep17.go
@@ -359,7 +359,10 @@ func multiTransferNEP17(ctx *cli.Context) error {
 	defer wall.Close()
 
 	fromFlag := ctx.Generic("from").(*flags.Address)
-	from := fromFlag.Uint160()
+	from, err := getDefaultAddress(fromFlag, wall)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
 	acc, err := getDecryptedAccount(ctx, wall, from)
 	if err != nil {
 		return cli.NewExitError(err, 1)
@@ -422,7 +425,10 @@ func transferNEP17(ctx *cli.Context) error {
 	defer wall.Close()
 
 	fromFlag := ctx.Generic("from").(*flags.Address)
-	from := fromFlag.Uint160()
+	from, err := getDefaultAddress(fromFlag, wall)
+	if err != nil {
+		return cli.NewExitError(err, 1)
+	}
 	acc, err := getDecryptedAccount(ctx, wall, from)
 	if err != nil {
 		return cli.NewExitError(err, 1)
@@ -483,4 +489,15 @@ func signAndSendTransfer(ctx *cli.Context, c *client.Client, acc *wallet.Account
 
 	fmt.Fprintln(ctx.App.Writer, tx.Hash().StringLE())
 	return nil
+}
+
+func getDefaultAddress(fromFlag *flags.Address, w *wallet.Wallet) (util.Uint160, error) {
+	if fromFlag.IsSet {
+		return fromFlag.Uint160(), nil
+	}
+	addr := w.GetChangeAddress()
+	if addr.Equals(util.Uint160{}) {
+		return util.Uint160{}, errors.New("can't get default address")
+	}
+	return addr, nil
 }

